bip-0085.mediawiki

```diff
@@ -360,36 +360,58 @@ OUTPUT
 * DERIVED ENTROPY=f7cfe56f63dca2490f65fcbf9ee63dcd85d18f751b6b5e1c1b8733af6459c904a75e82b4a22efff9b9e69de2144b293aa8714319a054b6cb55826a8e51425209
 * DERIVED PWD=_s`{TW89)i4`
 
-===RSA===
+===GPG Keys===
 
-Application number: 828365'
+Application number is dependant on the key type.
```

Comment:
If these are all GPG keys why not just add a segment for the key type? That would be more consistent with the rest of 85 so:
```
{one app number for all}/{key_type}/{bits}/{index}
```